README.md

```diff
@@ -1,4 +1,4 @@
-# CHAOSS Value Working Group
+# OSPO Metrics Working Group
```

Comment:
The name should be "CHAOSS OSPO Metrics Working Group" following the naming convention of all the working groups.